internal/kclvm_py/scripts/release.sh

```diff
@@ -1,17 +1,32 @@
 #!/usr/bin/env bash
 
+kclvm_source_dir="$topdir/internal/kclvm_py"
```

Comment:
There is a problem with the release script test, and the compressed package is wrong.